scout/server/blueprints/variants/templates/variants/fusion-variants.html

```diff
@@ -59,7 +59,8 @@
             <th style="width:3%"></th>
             <th style="width:4%" title="Beta version. Currently testing">Rank (beta)</th>
             <th style="width:6%" title="Genes">Fusion Genes</th>
-            <th style="width:4%" title="Number of callers detecting a given event">Callers</th>
+            <th style="width:4%" title="Number of callers detecting a given event">Nr. callers</th>
+            <th style="width:4%" title="Database hits">Database hits</th>
```

Comment:
I don't know, perhaps there is a better name than Database hits

Reply:
some additional ideas: Fusion Databases, Database Presence/Matches

Reply:
The SNVs and SVs pages use the general "Observed" for external db matches.

Reply:
Ah, right! Let's use the same key then!

Reply:
Fixed, thanks!